packages/teleport/src/Databases/Databases.tsx

```diff
@@ -33,12 +35,27 @@ export default function Container() {
 }
 
 export function Databases(props: Props) {
```

Comment:
forgot to mention in earlier PRs that Props from custom hooks should be named `State`